### DISCUSSION TITLE
Custom LNPopupItem

### DISCUSSION THREAD

**Discussion by KangTaeHoon** [LeoNatan/LNPopupController#495] [General] [answered]
Can custom items be used in addition to the defined LNPopupItem?

**Comment by LeoNatan** [comment, marked as answer]
Hello,
I’m not sure what you are asking. LNPopupItem objects are used to feed data to the framework, mostly for the standard popup bar styles. If you want a custom bar, it’s possible. Look at https://github.com/LeoNatan/LNPopupController/blob/master/LNPopupController/LNPopupController/LNPopupCustomBarViewController.h
